src/Pagination/PaginationManipulator.php

```diff
@@ -236,16 +238,14 @@ protected static function countArgument(?int $defaultCount = null, ?int $maxCoun
     {
         $description = '"Limits number of fetched items.';
         if ($maxCount) {
-            $description .= ' Maximum allowed value: ' . $maxCount . '.';
+            $description .= " Maximum allowed value: {$maxCount}.";
         }
         $description .= "\"\n";
 
-        $definition = 'first: Int'
-            . (
-                $defaultCount
-                ? ' = ' . $defaultCount
-                : '!'
-            );
+        $definition = 'first: Int!';
```

Comment:
Interface field argument Account.files(first:) expects type Int but Company.files(first:) is type Int!.